### PR TITLE
Generate_package small changes

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by mr-karan** [comment]
marking it WIP

**Comment by Adrianzatreanu** [comment]
can i ask why its a wip? the tests are all passing, its juts a bug from appveyor that times out, its here for review :) i forgot to restart the tests thats all ..

**Comment by mr-karan** [comment]
I'm sorry @Adrianzatreanu

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/666/commits/e5046c4d4e08d9e40a3dbb6824f213a43c585a34

**Comment by hypothesist** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/666/commits/e5046c4d4e08d9e40a3dbb6824f213a43c585a34 beacuse as Lasse said, the commit message could be better (hadn't noticed it, sorry)

File: bears/generate_package.py [hunk 1]
```diff
@@ -40,9 +40,7 @@ def create_file_from_template(template_file, output_file, substitution_dict):
 def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
```
Comment (hypothesist) [inline bears/generate_package.py]: double backticks around the `__init__.py` in the commit body

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/666/commits/844dcb0fa1244c55530fe6297e935b7b2030e6c4

File: bears/generate_package.py (hunk 1 above)
Comment (hypothesist) [inline bears/generate_package.py]: Also there seem to be other changes apart from just the `__init__.py` one. Detail everything or break up in to separate commits

Comment (hypothesist) [inline bears/generate_package.py]: Preferably use the same commit (since they're related)

File: bears/setup.py.in [hunk 2]
```diff
@@ -21,10 +21,9 @@ if __name__ == "__main__":
               maintainers_emails=$MAINTAINERS_EMAILS,
               platforms=$PLATFORMS,
               license='$LICENSE',
-              packages=find_packages(exclude=["build.*"]),
+              packages=find_packages(),
```
Comment (hypothesist) [inline bears/setup.py.in]: Sorry, I haven't really followed this PR. What's this change about?

File: bears/generate_package.py (hunk 1 above)
Comment (Adrianzatreanu) [inline bears/generate_package.py]: these changes are all related to the init.py one, they all are related because changing this also changes entry points and also changes tests :)

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/666/commits/a2c372e85fb8396119ba44318cf973bb60005d2a

**Comment by Asnelchristian** [comment]
needs a rebase btw.

**Comment by Adrianzatreanu** [comment]
reack 9fedf19

**Comment by Adrianzatreanu** [comment]
reack 030db5628f6

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by Adrianzatreanu** [comment]
reack 94e0c93 956caa4

**Comment by Adrianzatreanu** [comment]
reack aca76ff

**Comment by Adrianzatreanu** [comment]
@rultor merge
